SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimAnalyzer.cc

```diff
@@ -141,21 +286,74 @@ EcalEBTrigPrimAnalyzer::analyze(const edm::Event& iEvent, const  edm::EventSetup
   }
 
 
-  //  if (!recHits_) return;
 
+  // Get GenParticles
+  edm::Handle<reco::GenParticleCollection> gp;
+  iEvent.getByToken(genPartToken_,gp);
+  isGenParticleValid_=false;
+  if ( gp.isValid() ) 
+    isGenParticleValid_=true;
+  if (isGenParticleValid_) std::cout << " Gen Particle size " << gp->size();
+
+  //
+  /*
```

Comment:
Delete commented out code, or justify with a comment why you still need to keep it